ctru-rs/src/services/ac.rs

```diff
@@ -0,0 +1,349 @@
+//! The Automatic Connection (AC) service handles Wi-Fi and network settings.
+//! It can:
+//! - Connect to a network or slot
+//! - Get information about a network, such as its SSID or security settings
+use crate::error::ResultCode;
+
+/// Handle to the Automatic Connection (AC) service, that handles Wi-Fi and network settings.
+pub struct Ac(());
+
+impl Ac {
+    /// Initialize a new service handle.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # let _runner = test_runner::GdbRunner::default();
+    /// # use std::error::Error;
+    /// # fn main() -> Result<(), Box<dyn Error>> {
+    /// #
+    /// use ctru::services::ac::Ac;
+    ///
+    /// let ac = Ac::new()?;
+    /// #
+    /// # Ok(())
+    /// # }
+    /// ```
+    #[doc(alias = "acInit")]
+    pub fn new() -> crate::Result<Ac> {
+        unsafe {
+            ResultCode(ctru_sys::acInit())?;
```

Comment:
Note: if we ever plan to implement methods such as setting up wifi connections, we would need a `&mut` reference and the `ServiceReference` structure to keep the thing thread-safe. For now it's not needed.